### PR TITLE
feat: add client-setup package

### PR DESCRIPTION
**Pull request by alaincormier-ssc**
new package for issue #370
(additional PRs will be created for other packages and to update documentation)

**`solutions/client-setup`**
- includes / deprecates packages:
    - none
- adds features:
    - client namespaces
    - client folder
    - client management project
    - client tier2 root sync
- depends on:
    - solutions/core-landing-zone

### CODE REVIEW

File: solutions/client-setup/Kptfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+metadata:
+  name: client-setup
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: |
+    Landing zone v2 subpackage.
+    Depends on core-landing-zone
+
+    Package to setup a client's namespaces, management project and root sync
```
Comment (davelanglois-ssc) [inline solutions/client-setup/Kptfile:12]: Should we add that it is also deploying the initial client folder

**Review by davelanglois-ssc — COMMENTED** [review]
just missing the release-please stuff

**Comment by alaincormier-ssc** [comment]
> just missing the release-please stuff

I was thinking to remove it from PR #374 for now and do a separate PR once all new packages are created to include all releases at that time.
It'll avoid dealing with merge conflicts and we could maybe also remove the old packages from the release-please configs at that time?

File: solutions/client-setup/Kptfile (hunk 1 above)
Comment (alaincormier-ssc) [reply solutions/client-setup/Kptfile]: done

**Review by davelanglois-ssc — DISMISSED** [review]

**Review by fmichaelobrien — APPROVED** [review]
LGTM

**Review by cartyc — APPROVED** [review]
/lgtm

**Pull request merged**